pkg/doc/verifiable/support_test.go

```diff
@@ -41,6 +41,7 @@ const validCredential = `{
   "@context": [
     "https://www.w3.org/2018/credentials/v1",
     "https://www.w3.org/2018/credentials/examples/v1",
+	"https://trustbloc.github.io/context/vc/credentials-v1.jsonld",
```

Comment:
Could you cache `https://trustbloc.github.io/context/vc/credentials-v1.jsonld` context for the VC tests? It's done at https://github.com/hyperledger/aries-framework-go/blob/c942c043873c23ccb7fe68bc177811efa89078d7/pkg/doc/verifiable/support_test.go#L366

Reply:
done